Tools/CISupport/ews-app/ews/views/submittoews.py

```diff
@@ -45,11 +45,17 @@ def get(self, request):
     def post(self, request):
         try:
             change_id = request.POST.get('change_id')
+            _log.info('SubmitToEWS::change: {}'.format(change_id))
             change_id = int(change_id)
         except:
             return HttpResponse('Invalid patch id provided, should be an integer. git hashes are not supported.')
 
-        _log.info('SubmitToEWS::change: {}'.format(change_id))
+        if change_id < 430000:
```

Comment:
430000 is around May-2021. We can adjust it if anyone feel it's not very appropriate.

Reply:
If it's possible to fetch the latest patch date, that would be more ideal since this blocks old bugs from being picked up again and tested within Bugzilla. (but since we want to deprecate this anyways, I am not sure)

Reply:
and by picked up I mean re-submitted as a patch to the same Bugzilla.

Reply:
It's possible but Alexey had another good point about ChangeLog. Went with that.